### DISCUSSION TITLE
Chatisfy 打造專屬的聊天機器人

### DISCUSSION THREAD

**Discussion by blackline0911** [1101-GS4719-C/Python-Community-GS4719-C#558] [General]
Chatisfy 可以打造專屬的聊天機器人
Chatisfy打造出來的聊天機器人與眾不同的地方是可以作為電子商務平台、下訂單、相當於購物平台的服務，適合企業發展推銷自己的產品使用，達到即時服務的效果
打造的方式不用寫程式語言，而是用類似LABVIEW圖像化語言的方式進行撰寫，適合未接觸過程式的人開發。
詳細介紹在此：
https://www.anyong.com.tw/stories/mkt-chatbot-choose-13512
開發網址在此：
https://www.chatisfy.com/

**Comment by JSHT** [comment]
透過在網站嵌入 Facebook Messenger，搭配第三方 chatbot 進行服務的設置真的是蠻方便的。

**Comment by chia-kai-chang** [comment]
我之前使用過 BotStar ( https://botstar.com/ )，也是用流程圖的方式來建立聊天機器人。

![image](https://user-images.githubusercontent.com/1768891/164482374-d9eae812-93b9-48c5-8f13-8c38f6fdc21b.png)

剛剛玩了一下GOSU BAR，覺得他的介面設計很厲害，要做到這樣的網頁真是不簡單，看起來是用Javascript手刻出來的。

![image](https://user-images.githubusercontent.com/1768891/164486287-5f40e78c-2486-4cdf-8374-0480d1832794.png)

**Comment by ThomasMeng0515** [comment]
雖然可以用像LabVIEW那樣圖形化去建構聊天機器人，但感覺還是得具備程式開發的基本概念才能成功建構機器人，不過這真的會是一個很方便的工具
